MakingCoffee/src/MakingCoffeeEx1.java

```diff
@@ -0,0 +1,48 @@
+
+import java.util.Scanner;
+public class MakingCoffeeEx1 {
+
+    public static void main(String[] args) {
+        int waterPerCup = 200;
```

Comment:
@Astgh78 jan please split method on small once as we discussed yesterday

Reply:
Nana jan, didn't understand which method you mean.